components/atoms/AlphaBanner.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Displays the Alpha AlphaBanner on the page
+ */
+
+export const AlphaBanner = ({ text }) => {
+  return (
+    <div className="bg-gray-100">
```

Comment:
banner too thick, in the mockups on figma banner is 55px in height

Reply:
With tailwind, the closest I can get is 56px.